cloud/resource_cloud_environment.go

```diff
@@ -70,11 +67,11 @@ func resourceCloudEnvironment() *schema.Resource {
 				Description:  descriptions["organization"],
 				ValidateFunc: validateNotBlank,
 			},
-			"name": {
+			"type": {
 				Type:         schema.TypeString,
 				Required:     true,
-				Description:  descriptions["cloud_environment_name"],
-				ValidateFunc: validateCloudEnvionmentName,
+				Description:  descriptions["cloud_environment_type"],
+				ValidateFunc: validateCloudEnvionmentType,
```

Comment:
let's unify the param named `environment_type` as the same as snctl

Reply:
Not entirely sure what you mean by this

Reply:
+1